decoimpact/data/entities/data_access_layer.py

```diff
@@ -141,6 +141,10 @@ def write_output_file(
         """
         self._logger.log_info(f"Writing model output data to {path}")
 
+        if not Path.exists(path.parent):
+            # try to make intermediate folders
+            Path(path.parent).mkdir(parents=True, exist_ok=True)
+
         if not Path.exists(path.parent):
```

Comment:
this can be deleted now right? unreachable code?

Reply:
Not sure, I kept it to be on the save side. For example, if you lack permissions to create a folder you can still end up there

Reply:
ok :)